### DISCUSSION TITLE
Beaten Up by Mime Executives

### DISCUSSION THREAD

**Discussion by DrBaudais** [loathers/garbage-collector#2019] [Q&A] [answered]
Now that Garbo is cloning the Mime Executives, I am consistently getting beat up by them due to garbo getting the "Let's go Shopping!" buff, which sets my myst to 1. It then waits until turn 27 to start attacking, with spells (Sauce and due to low myst I'm not getting the kill. I think part of the problem is that I'm a Pastamancer. I'm wondering if there is something I can do to resolve this without having to ascend. More spell dmg? Missing a crucial damaging spell?

**Comment by btven** [comment, marked as answer]
As a short-term fix I just use a couple of big +myst% potions on my multi and that seems to get him through those combats. Think pressurized potion and other similar options.

**Reply by DrBaudais** [reply]
That makes a lot of sense... It didn't even occur to me that the debuff is -300%, so I just need to get back into positives. Thank you!
Is there a long-term solution? I'm realizing that musc/mox class might not even solve my issue since it's a scaling monster.

**Comment by btven** [comment]
I'll leave it to the garbo devs to determine whether there is a long term solution or not. I've only encountered this on my multi which runs garbo as a sauceror. On my main, which I have been running as a seal clubber, I have not had this issue. I'm not sure if it's simply a difference in class/combat strategy or whether my main is sufficiently shiny to not have this problem.
